### PR TITLE
Created symlink for logging

### PR DESCRIPTION
**Pull request by leahnp**
Added rkt flags to create symlink for docker logging, links var/lib/docker/container/logs to var/log/containers

### CODE REVIEW

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
dummy comment, testing notifications

**Comment by coffeepac** [comment]
LGTM

File: ansible/roles/kraken.node/kraken.node.docker/templates/units.kubelet.part.jinja2 [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
     ExecStartPre=/usr/bin/mkdir -p /var/lib/kubelet
+    ExecStartPre=/usr/bin/mkdir -p /var/log/containers
     ExecStart=/usr/bin/rkt run \
       --volume etc-resolv-conf,kind=host,source=/etc/resolv.conf \
       --volume etc-cni,kind=host,source=/etc/cni \
       --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
       --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
       --volume var-lib-docker,kind=host,source=/var/lib/docker \
       --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
+      --volume var-log,kind=host,source=/var/log \
```
Comment (maratoid) [inline ansible/roles/kraken.node/kraken.node.docker/templates/units.kubelet.part.jinja2]: Should we also mount a new EBS volume for /var/log like we do for , for example /var/lib/docker (https://github.com/samsung-cnct/k2/blob/master/ansible/roles/kraken.config/files/config.yaml#L141) ?

Comment (maratoid) [inline ansible/roles/kraken.node/kraken.node.docker/templates/units.kubelet.part.jinja2]: Should we also mount a new EBS volume for /var/log like we do for , for example /var/lib/docker (https://github.com/samsung-cnct/k2/blob/master/ansible/roles/kraken.config/files/config.yaml#L141) ?

**Pull request merged**